Tutorial_1c_Line_Ratios.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<p>I think it's worth giving a bit more explanation on the geometry that's being assumed here, just to break up the projection steps in the code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>